app/lib/general.lib.js

```diff
@@ -98,6 +98,41 @@ function determineCurrentFinancialYear () {
   return { startDate: new Date(startYear, 3, 1), endDate: new Date(endYear, 2, 31) }
 }
 
+/**
+ * Generate a string that represents an abstraction point
+ *
+ * When abstracting water the point at which this is done can be described in several ways depending on the number of
+ * Nation Grid References are saved against the abstraction point. This function checks for these references and builds
+ * a string that defines the details of the abstraction point.
+ *
+ * @returns {String} a description of the abstraction point
```

Comment:
Just missing the param

```suggestion
 * a string that defines the details of the abstraction point.
 *
 * @param {Object} pointDetail - Object containing all the details for the point 
 *
 * @returns {String} a description of the abstraction point
```